### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by nhsz** [ethereum/ethereum-org-fork#9]
### Describe the bug

We have some issues related to the styles of `DocLink` and `ButtonLink` components

### To reproduce

- for `DocLink` check `/community/support` page
- for `ButtonLink` check `/guides/how-to-swap-tokens` page

### Expected behavior

Proper styling

### Screenshots

DocLink
<img width="802" alt="Screen Shot 2023-08-16 at 15 49 55" src="https://github.com/ethereum/ethereum-org-fork/assets/948922/aaceb125-3aca-4163-ac77-440fee337728">

ButtonLink
<img width="795" alt="Screen Shot 2023-08-16 at 15 50 09" src="https://github.com/ethereum/ethereum-org-fork/assets/948922/969d1f4f-d57a-4735-90fc-08a40eae1968">

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

**Comment by pettinarip** [comment]
This problem is related to the new version of mdx that we are using here, v2.

The current syntax that we are using works with mdx v1. Since we have upgraded it to v2, a few things have changed. For example, content that is inside jsx is now considered markdown, so:

```mdx
<Button>
  See our other guides
</Button>

// in v1, this is converted to:
<button>
  See our other guides
</button>

// in v2, the new line will means that this is a new paragraph:
<button>
  <p>See our other guides</p>
</button>
```

Then what we should do to keep the previous form in mdx v2, would be:
```mdx
// avoid the new line
<Button>See our other guides</Button>

// or we could use this form to avoid being parsed as markdown
// https://mdxjs.com/migrating/v2/#expressions
{
<Button>
  See our other guides
</Button>
}
```

More about changes in v2: https://mdxjs.com/migrating/v2/#update-mdx-files


-----

IMO given that fixing this issue would imply changes in a lot of md pages (and a lot of sync conflicts with upstream), I'd propose not doing this mdx migration now and creating its own epic/task to be tackled after we finish the current NextJS migration.

This would mean to downgrade mdx to v1, which means that we'd need to downgrade `next-mdx-remote` to v3.

cc @nhsz @corwintines @wackerow @minimalsm

**Comment by pettinarip** [comment]
If we all agree then I will reopen https://github.com/ethereum/ethereum-org-website/issues/9160 to keep track of that.

**Comment by nhsz** [comment]
> IMO given that fixing this issue would imply changes in a lot of md pages (and a lot of sync conflicts with upstream), I'd propose not doing this mdx migration now and creating its own epic/task to be tackled after we finish the current NextJS migration.

@pettinarip thanks for the research. Agree to downgrade `next-mdx-remote` to v3 for now and migrate to mdx v2 later. We would need to test that `next-mdx-remote` v3 doesnt break the build or introduce other bugs

**Comment by pettinarip** [comment]
@nhsz cool, I can tackle that. Will create a new card.

On the other hand, I'll reopen the [mentioned issue](https://github.com/ethereum/ethereum-org-website/issues/9160).

**Comment by nhsz** [comment]
@pettinarip re-opening as there's still a minor difference with the styling of `DocLink` component, maybe related to the updated `Link` component

now rendering 

<img width="815" alt="Screen Shot 2023-09-05 at 10 49 34" src="https://github.com/ethereum/ethereum-org-fork/assets/948922/c413235d-2edd-4a11-b8ed-0e5d88739d4a">

---

should be 

<img width="822" alt="Screen Shot 2023-09-05 at 10 51 59" src="https://github.com/ethereum/ethereum-org-fork/assets/948922/37a23968-45f7-4696-8ce1-082d6bdc719f">

**Comment by nhsz** [comment]
Fixed now by #21, thx!
